docs/gettingstarted.rst

```diff
@@ -10,6 +10,16 @@ find the latest release at https://github.com/VirusTotal/yara/releases.
 Compiling and installing YARA
 =============================
 
+You can download and install YARA using the `vcpkg <https://github.com/Microsoft/vcpkg/>`_ dependency manager::
+
+    git clone https://github.com/microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    vcpkg install yara
+
+YARA is kept up to date by Microsoft team members and community contributors. If the version of YARA in vcpkg is out of date, please `create an issue or pull request <https://github.com/Microsoft/vcpkg/>`_ on the vcpkg repository.
```

Comment:
Shouldn't this say "The vcpkg is kept up to date by Microsoft team members and community contributors."

Reply:
Yes, it seems to say that YARA as a whole is maintained by Microsoft.

Reply:
Fixed! Sorry for the poor wording. Thanks for the feedback @wxsBSD and @plusvic. Let me know if there's anything else